### PR TITLE
WMS Service Catalog sample

### PR DESCRIPTION
**Pull request by nCastle1**

### CODE REVIEW

**Comment by nCastle1** [comment]
@ThadT Do you have any capacity to review a sample?

**Comment by ThadT** [comment]
@nCastle1 Sure, sign me up!

**Comment by ThadT** [comment]
Hey @nCastle1 
Sorry for the delay, I had some computer issues to work through. It looks like I'm getting an error when running on WPF or UWP. iOS works (native and forms) and I haven't tried Android yet.

File: src/Android/Xamarin.Android/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.cs [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+namespace ArcGISRuntimeXamarin.Samples.WmsServiceCatalog
+{
+    [Activity]
+    public class WmsServiceCatalog : Activity
+    {
+        // Create and hold reference to the used MapView
+        private MapView _myMapView = new MapView();
+
+        // Hold a reference to the listview
```
Comment (ThadT) [inline src/Android/Xamarin.Android/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.cs]: 'ListView'

File: src/Android/Xamarin.Android/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.cs [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+        private void CreateLayout()
+        {
+            // Create a new vertical layout for the app
+            var layout = new LinearLayout(this) { Orientation = Orientation.Vertical };
+
+            // Create the list view
+            _myDisplayList = new ListView(this);
+
+            // Add the listview and mapview to the layout
```
Comment (ThadT) [inline src/Android/Xamarin.Android/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.cs]: ListView, MapView

File: src/Android/Xamarin.Android/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.cs [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+                _viewModel.Add(new LayerDisplayVM(layerInfo));
+            }
+
+            // Create an array adapter for the layer display
+            ArrayAdapter adapter = new ArrayAdapter(this, Android.Resource.Layout.SimpleSpinnerItem, _viewModel);
+
+            // Apply the adapter
+            _myDisplayList.Adapter = adapter;
+
+            // Subscribe to seleciton change notifications
```
Comment (ThadT) [inline src/Android/Xamarin.Android/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.cs]: 'selection'

File: src/iOS/Xamarin.iOS/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.cs [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+    [Register("WmsServiceCatalog")]
+    public class WmsServiceCatalog : UIViewController
+    {
+        // Create and hold reference to the used MapView
+        private MapView _myMapView = new MapView();
+
+        // Hold the URL to the WMS service providing the US NOAA National Weather Service forecast weather chart
+        private Uri wmsUrl = new Uri("https://idpgis.ncep.noaa.gov/arcgis/services/NWS_Forecasts_Guidance_Warnings/natl_fcst_wx_chart/MapServer/WMSServer?request=GetCapabilities&service=WMS");
+
+        // Hold a source for the tableview that shows the available WMS layers
```
Comment (ThadT) [inline src/iOS/Xamarin.iOS/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.cs]: 'table view' or 'UITableView'

File: src/iOS/Xamarin.iOS/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.cs [hunk 5]
```diff
@@ -0,0 +1,267 @@
...
+            // Create the UI, setup the control references
+            CreateLayout();
+
+            // Initialize the map
+            Initialize();
+        }
+
+        private void CreateLayout()
+        {
+            // Add the mapview to the view
```
Comment (ThadT) [inline src/iOS/Xamarin.iOS/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.cs]: MapView

**Pull request merged**